db/migrate/20200120203124_add_parent_profile_id_to_profiles.rb

```diff
@@ -0,0 +1,5 @@
+class AddParentProfileIdToProfiles < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :profiles, :parent_profile, type: :uuid, foreign_key: { to_table: :profiles }
```

Comment:
Probably just `parent` is enough (seems a bit repetitive "profile.parent_profile")

Reply:
First, I think `parent` and `child` are both generic to tree structures. While it may seem repetitive, I think being explicit about the parent type here will help keep our code and API more readable. A profile might have other parents of different types now or in the future. One might say a non-canonical profile has a parent profile and a parent benchmark, as XML is all a big tree structure. I also wondered at first if we should call this `extends`, as that is what the XCCDF spec calls it, but I also think that name is not very descriptive, mostly because it leaves off the type (profile).